src/App.test.js

```diff
@@ -1,8 +1,28 @@
+import React from 'react';
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import renderer from 'react-test-renderer';
+import Calculator from './components/Calculator';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+describe('All components', () => {
+  test('renders Quote component', () => {
+    const h2 = renderer.create(<h2>New Quote</h2>).toJSON();
+    expect(h2).toMatchInlineSnapshot(`
+<h2>
```

Comment:
- [ ] Currently, I am not able to see your components snapshots I suggest that instead of using `toMatchInlineSnapshot` use `toMatchSnapshot` all will be fine